plugin_store/database/database.py

```diff
@@ -162,6 +179,15 @@ async def search(
                 statement = statement.filter(Artifact.tags.any(tag=tag))
         if not include_hidden:
             statement = statement.where(Artifact.visible.is_(True))
+        direction = desc()
+        match (order_by):
+            case SortOption.asc: direction = asc()
+            case SortOption.desc: direction = desc()
+            case SortOption.NONE: pass
+        match (sort_by):
+            case SortType.name: statement = statement.order_by(direction(Artifact.name))
+            case SortType.date: statement = statement.order_by(direction(Artifact.created))
+            case SortType.NONE: pass
```

Comment:
We're still using python 3.9, match is not available.

Reply:
That's a shame. I'll switch it to if else then :(